src/main/java/http/enums/Method.java

```diff
@@ -0,0 +1,11 @@
+package http.enums;
+
+public enum Method {
+    GET("GET") , POST("POST");
+
+    String name;
```

Comment:
현재 선언된 name 을 사용하는 부분이 없는 것 같아요.
java enum 의 valueOf 메서드만 사용한다면, 아래와 같이 수정해도 무방할 것 같습니다!
```suggestion
    GET , POST;

    String name;
```